src/molecules/product-list-item/ProductListItem.html

```diff
@@ -0,0 +1,128 @@
+      <slot
+        name="name"
+        :product-name="name"
+        :product-url="url"
+      >
+        <router-link
+          :to="url"
+          class="link link--inverted a-product-list-item__name"
+        >
+          {{ name }}
+        </router-link>
+      </slot>
+      <!--@slot Custom product price (Scoped slot)-->
+      <slot
+        name="price"
+        :price="price"
+        :special-price="specialPrice"
+        :old-price="oldPrice"
+      >
+        <a-price
+          :price="price"
+          :special-price="specialPrice"
+          :old-price="oldPrice"
+          class="a-product-list-item__price-wrapper"
+          old-price-class="a-product-list-item__price-wrapper--old"
+          special-price-class="a-product-list-item__price-wrapper--special"
+        />
+      </slot>
+
+      <a-list
+        v-if="options"
+        tag="dl"
+        class="a-product-list-item__attributes"
+      >
+        <template v-for="item in options">
+          <!-- @slot Custom product item option (Scoped slot) -->
+          <slot
+            name="option"
+            :option="item"
+          >
+            <a-list-item tag="dt">
+              {{ item.label }}
+            </a-list-item>
+            <a-list-item tag="dd">
+              {{ item.value }}
+            </a-list-item>
+          </slot>
+        </template>
+      </a-list>
+    </div>
+
+    <div
+      :class="[
+        'a-product-list-item__actions',
+        { 'a-product-list-item__actions--to-right': hideQty }
+      ]"
+    >
+      <div
+        v-if="!hideQty"
+        class="a-product-list-item__qty-wrapper"
+      >
+        <!--@slot Custom product quantity (Scoped slot)-->
+        <slot
+          name="quantity"
+          :item-qty="qty"
+          :change-qty="onChange"
+        >
+          <a-quantity-update
+            :value="qty"
+            input-id="qty_1"
+            class="product-list-item__qty"
+            @update="onChange"
+            decrement-aria-label="Decrease the quantity"
+            decrement-icon-title="Minus mark"
+            increment-aria-label="Increase the quantity"
+            increment-icon-title="Plus mark"
+          >
+            <template #label>
+              <a-label
+                class="product-list-item__qty-label"
+                for="qty_1"
+              >
+                Qty:
+              </a-label>
+            </template>
+          </a-quantity-update>
+        </slot>
+      </div>
+
+      <div class="a-product-list-item__action-buttons-wrapper">
+        <!--@slot Custom product remove button (Scoped slot)-->
+        <slot
+          name="removeAction"
+          :item-remove="onRemove"
+        >
+          <a-button
+            class="a-product-list-item__action-button"
+            aria-label="Remove this product from your shopping cart"
+            custom-class="a-product-list-item__actions-item"
+            @click.native="onRemove"
+          >
+            <a-icon
```

Comment:
wrong icon color

Reply:
It's hard to get it from zeplin (also there is no clue in svg file) but imo this is base color with changed opacity. Styles updated, let me know if you would like to add new color here.